test/e2e/test_image.py

```diff
@@ -125,7 +125,13 @@ def test_create_with_file_extension_content_type_mismatch(self):
         self.post_image_with_file_extension_content_type_mismatch(
             IMAGE_POST_METADATA_DATA_REQUIRED_VALUES_ONLY, "image.jpg"
         )
-        self.check_post_image_failed_with_detail(422, "Filename does not contain the correct extension")
+        self.check_post_image_failed_with_detail(422, "File extension and content type do not match")
+
+    def test_create_with_unsupported_file_extension(self):
+        """Test creating an image with an unsupported file extension."""
+
+        self.post_image(IMAGE_POST_METADATA_DATA_REQUIRED_VALUES_ONLY, "image.webp")
```

Comment:
why test with an extension that would be allowed according to `object_storage_api/.env.example`? Wouldn't it make more sense to use an extension that is completely not allowed (like .gif). Or is there a reason for doing this (like to test it works when a usually supported extension isn't exported in this env)?

Reply:
It's only an example so the actual file doesn't necessarily need to support it. Also depending on needs `.gif` might be allowed so. It was simpler and quicker at the time for me to just convert the existing `image.jpg` to `image.webp`.

I can look into ways of creating a `.gif` image and using that over `.webp` if others also think the same.

Reply:
Ok, well I'm happy for it to stay as `.webp` then, unless someone else thinks it should change.